test/Microsoft.TestPlatform.AcceptanceTests/ExecutionTests.cs

```diff
@@ -16,21 +16,26 @@ namespace Microsoft.TestPlatform.AcceptanceTests;
 [TestClass]
 public class ExecutionTests : AcceptanceTestBase
 {
+    // TODO: It looks like the first 3 tests would be useful to multiply by all 3 test frameworks, should we make the test even more generic, or duplicate them?
+
     [TestMethod]
-    [NetFullTargetFrameworkDataSource(inIsolation: true, inProcess: true)]
-    [NetCoreTargetFrameworkDataSource]
-    public void RunMultipleTestAssemblies(RunnerInfo runnerInfo)
+    [MSTestCompatibilityDataSource(InProcess = true)]
+    public void RunMultipleTestAssemblies(RunnerInfo runnerInfo, MSTestInfo msTestInfo)
     {
         SetTestEnvironment(_testEnvironment, runnerInfo);
 
-        var assemblyPaths = BuildMultipleAssemblyPath("SimpleTestProject.dll", "SimpleTestProject2.dll").Trim('\"');
+        var assemblyPaths = BuildMultipleAssemblyPath(msTestInfo, "SimpleTestProject.dll", "SimpleTestProject2.dll").Trim('\"');
```

Comment:
Just doing it this way kinda sucks, but I want to get it working with what I have now, and minimal changes, and then review the existing patterns.